packages/patternfly-4/react-core/src/layouts/Flex/FlexUtils.tsx

```diff
@@ -2,98 +2,112 @@ import * as React from 'react';
 import { getModifier } from '@patternfly/react-styles';
 import styles from '@patternfly/react-styles/css/layouts/Flex/flex';
 
+export enum FlexModifiers {
+  flex = 'flex',
+  'inline-flex' = 'inline-flex',
+  'spacer-none' = 'spacer-none',
+  'spacer-xs' = 'spacer-xs',
+  'spacer-sm' = 'spacer-sm',
+  'spacer-md' = 'spacer-md',
+  'spacer-lg' = 'spacer-lg',
+  'spacer-xl' = 'spacer-xl',
+  'spacer-2xl' = 'spacer-2xl',
+  'spacer-3xl' = 'spacer-3xl',
+  'space-items-none' = 'space-items-none',
+  'space-items-xs' = 'space-items-xs',
+  'space-items-sm' = 'space-items-sm',
+  'space-items-md' = 'space-items-md',
+  'space-items-lg' = 'space-items-lg',
+  'space-items-xl' = 'space-items-xl',
+  'space-items-2xl' = 'space-items-2xl',
+  'space-items-3xl' = 'space-items-3xl',
+  grow = 'grow',
+  shrink = 'shrink',
+  'flex-1' = 'flex-1',
+  'flex-2' = 'flex-2',
+  'flex-3' = 'flex-3',
+  'flex-4' = 'flex-4',
+  'flex-default' = 'flex-default',
+  'flex-none' = 'flex-none',
+  column = 'column',
+  'column-reverse' = 'column-reverse',
+  row = 'row',
+  'row-reverse' = 'row-reverse',
+  wrap = 'wrap',
+  'wrap-reverse' = 'wrap-reverse',
+  nowrap = 'nowrap',
+  'align-right' = 'align-right',
+  'align-left' = 'align-left',
+  'align-self-flex-start' = 'align-self-flex-start',
+  'align-self-flex-end' = 'align-self-flex-end',
+  'align-self-flex-center' = 'align-self-flex-center',
+  'align-self-flex-baseline' = 'align-self-flex-baseline',
+  'align-self-flex-stretch' = 'align-self-flex-stretch',
+  'justify-content-flex-end' = 'justify-content-flex-end',
+  'justify-content-center' = 'justify-content-center',
+  'justify-content-flex-space-between' = 'justify-content-flex-space-between',
```

Comment:
@tlabaj Not sure if this value is correct. Docs example says to use `justify-content-space-between` https://github.com/patternfly/patternfly-react/pull/3258/files#diff-fee6334c94230e770989b535c35f87f7R428
Also, `justify-content-flex-space-between` doesn't work when this is used an undefined class is applied to the element.

Reply:
:fearful: ho snap! I see this now. It looks like the Modifier type was in correct to begin with. It didn't have `justify-content-space-between` and the flex.css doesn't have `justify-content-space-between` but `justify-content-space-between`.

My mistake for not paying attention. I will suggest a fix